src/app/pages/Comment.js

```diff
@@ -0,0 +1,25 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import CommentView from '../components/Comment';
+
+const Comment = props => {
+  const [commentState, setCommentState] = useState([]);
```

Comment:
```suggestion
  const [commentState, setCommentState] = useState(null);
```
лучше таким образом записать, так как в противном случае `commentState` всегда будет `truthy` (т. е. всегда будет переводиться в `true`, даже если это пустой массив):
`{commentState && <CommentView {...commentState} />}`